src/NuGet.Clients/NuGet.VisualStudio.Common/UserInterfaceService/INuGetUILogger.cs

```diff
@@ -1,22 +1,29 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Threading.Tasks;
 using NuGet.Common;
 
 namespace NuGet.PackageManagement.VisualStudio
 {
     public interface INuGetUILogger
     {
+        void Start();
+        Task StartAsync();
```

Comment:
From an API design point of view, `Start` and `End` on a logging class doesn't makr a lot of sense to me. But I see that these methods already existed in the implementation.